ansible/roles/polkadot-restore-db/tasks/main.yml

```diff
@@ -4,13 +4,23 @@
     state: present
     update_cache: yes
 
+- name: db_dest_path as fact
+  set_fact:
+    db_dest_path: "{{ base-path }}/chains/{{ polkadot_network_id }}"
```

Comment:
The substrate node will create the `chains/<network>` directory automatically when you just specify `base-path` Have you tested this?

Reply:
Let me show you this example:
![image](https://user-images.githubusercontent.com/25957738/111661216-20233f80-880f-11eb-9f84-b3e094095ba1.png)